wire-ios-sync-engine/Source/UserSession/Search/SearchDirectory.swift

```diff
@@ -52,6 +52,25 @@ import Foundation
         return task
     }
 
+    public func refetchIncompleteUserMetadata() {
+        let managedObjectContext = contextProvider.viewContext
```

Comment:
Is there a specific reason to do this on the view context?

Reply:
This appears to duplicate code in the timer manager class.

Reply:
because we are on the main thread. Updated this code